docs/azure/integrations-overview.md

```diff
@@ -1,15 +1,39 @@
 ---
 title: Azure integrations overview
 description: Overview of the Azure integrations available in the .NET Aspire.
-ms.date: 12/17/2024
+ms.date: 02/20/2025
 uid: dotnet/aspire/integrations/azure-overview
 ---
 
 # .NET Aspire Azure integrations overview
 
 [Azure](/azure) is the most popular cloud platform for building and deploying [.NET applications](/dotnet/azure). The [Azure SDK for .NET](/dotnet/azure/sdk/azure-sdk-for-dotnet) allows for easy management and use of Azure services. .NET Aspire provides a set of integrations with Azure services, where you're free to add new resources or connect to existing ones. This article details some common aspects of all Azure integrations in .NET Aspire and aims to help you understand how to use them.
 
-## Add connection to existing Azure resources
+## Naming conventions and APIs for expressing Azure resources
+
+The distributed application builder, part of the [app host](../fundamentals/app-host-overview.md), employs the builder pattern, allowing consumers to `AddAzure*` resources to the [_app model_](../fundamentals/app-host-overview.md#terminology). In addition to simply adding resources, developers can also express configuration and how said resource behave in various execution contexts. Azure hosting integrations provide APIs to specify how these resources should be "published" and "run".
+
+### App host run modes
+
+When the app host executes, the [_execution context_](../fundamentals/app-host-overview.md#execution-context) determines whether the app host is in <xref:Aspire.Hosting.DistributedApplicationOperation.Run> or <xref:Aspire.Hosting.DistributedApplicationOperation.Publish> mode. The naming conventions for these APIs indicate the intended action for the resource. For more information on execution modes, see [Execution context](../fundamentals/app-host-overview.md#execution-context).
+
+The following table summarizes the naming conventions used to express Azure resources:
+
+| API | Description |
+|--|--|
+| `PublishAsConnectionString` | Changes the resource to be published as a connection string reference in the manifest. |
+| `PublishAsContainer` | Changes the resource to be published as a container in the manifest. |
```

Comment:
```suggestion
```

No one uses this method. I think it was a holdover from 8.0 work.

@mitchdenny @davidfowl - should we obsolete this API?

Reply:
This repo is using it: https://grep.app/search?f.repo=microsoft%2Fkernel-memory&q=PublishAsContainer

Reply:
> This repo is using it: https://grep.app/search?f.repo=microsoft%2Fkernel-memory&q=PublishAsContainer

Not against Azure resources. It is only using it against `.AddContainer`, and IMO it is redundant.

Reply:
I can remove it from the table.